src/Raven.Studio/typescript/common/generalUtils.ts

```diff
@@ -231,6 +231,15 @@ class genUtils {
 
         return input;
     }
+    
+    static sortUTC(timeStampA: string, timeStampB: string, mode: sortMode = "asc"): number {
```

Comment:
can we put extra column and sort using numbers? Parsing dates on every comparison is not good.

Reply:
done